### PR TITLE
use curl only - replace any wget

### PR DESCRIPTION
**Pull request by wmark**
Avoids requiring both tools.

Please note that no changes have been made to examples or travis*,
because those are not our actual build environment.

### CODE REVIEW

File: scripts/dgr-update.sh [hunk 1]
```diff
@@ -29,8 +29,7 @@ if [ -f "$dir/.last_dgr" ]; then
 fi
 
 if [ "$version" != "$last_dgr" ]; then
-	wget -O ${dir}/dgr.tar.gz $url
-	tar xvzf ${dir}/dgr.tar.gz --strip=1 -C ${dir}
-	rm ${dir}/dgr.tar.gz
-	echo $version > $dir/.last_dgr
+  curl --fail --silent --show-error --location "${url}" \
+  | tar --strip=1 -C ${dir}/ -xzv
+  printf "$version" >$dir/.last_dgr
```
Comment (wmark) [inline scripts/dgr-update.sh]: https://google.github.io/styleguide/shell.xml#Indentation

**Comment by wmark** [comment]
I highly recommend you switch off Travis. They use horribly outdated build environments. Please consider utilizing [Semaphore CI](https://semaphoreci.com/), for example.

https://travis-ci.org/blablacar/dgr/builds/179736520#L424

https://github.com/curl/curl/blob/b8c35f40f99cd1965d6e3152fc01071b73c3d2f1/docs/cmdline-opts/next.d#L5

**Comment by n0rad** [comment]
it's merged but github don't see it

**Comment by wmark** [comment]
Nice! By the way, *curl* works very well with a single file `/etc/ssl/certs/ca-certificates.crt` (path may vary). The per-CA ones and hashes in `/etc/ssl/certs` need not be shipped anymore. Removing them makes comparing *builder* images more of a pleasure to compare.

**Pull request closed without merging**